docs/setup.md

```diff
@@ -132,4 +132,4 @@ To use `treat` in a `gatsby` project, install `gatsby-plugin-treat` and add it t
 module.exports = {
   plugins: [`gatsby-plugin-treat`],
 };
-```
\ No newline at end of file
+```
```

Comment:
This is a trailing line break due to Prettier. Not sure why that wasn't there already.